ehcache-impl/src/main/java/org/ehcache/impl/internal/events/CacheEventDispatcherFactoryImpl.java

```diff
@@ -67,11 +67,13 @@ public <K, V> CacheEventDispatcher<K, V> createCacheEventDispatcher(Store<K, V>
     if (config != null) {
       threadPoolAlias = config.getThreadPoolAlias();
     }
-
-    ExecutorService orderedExecutor = executionService.getOrderedExecutor(threadPoolAlias, new LinkedBlockingQueue<>());
-    ExecutorService unOrderedExecutor = executionService.getUnorderedExecutor(threadPoolAlias, new LinkedBlockingQueue<>());
-
-    return new CacheEventDispatcherImpl<>(unOrderedExecutor, orderedExecutor);
+    try {
```

Comment:
This code looks fine... but after discussion with @cljohnso we have some concerns with what this issue is asking for. We're not convinced that the benefits of this behavior change are worth the amount of code change that would result. We're also wondering if the idea of a configuration the lacks a default thread pool makes sense.

Put another way, in the following configuration is the "bug" in:
1. the lack of a defined thread pool for the event listener, or
2. the lack of a default thread pool?
```xml
<config xmlns='http://www.ehcache.org/v3'>
    <thread-pools>
        <thread-pool alias="thread-pool" min-size="1" max-size="3"/>
    </thread-pools>

    <cache alias="cache">
        <key-type>java.lang.String</key-type>
        <value-type>java.lang.String</value-type>
        <heap>1000</heap>
    </cache-template>
</config>
```

My feeling right now is that the lack of a default thread-pool is more likely to be the bug. If that's true, we should be working on highlighting that problem, so that the user fixes that bug.